python/lsst/source/injection/inject_base.py

```diff
@@ -117,6 +117,16 @@ class BaseInjectConfig(PipelineTaskConfig, pipelineConnections=BaseInjectConnect
         doc="String to prefix to the entries in the *col_stamp* column, for example, a directory path.",
         default="",
     )
```

Comment:
Perhaps it would also make sense here to add a `recompute_variance` (or similar, better name) to allow the user to switch off the new variance plane functionality if they so wish. This would allow us to maintain backwards compatibility with older images produced prior to this PR.

Reply:
Keen to hear your thoughts on my original comment here, namely, adding in a `recompute_variance` config flag (or similar name) to allow the user to switch off variance plane modification if so desired.

Reply:
This sounds good - I've lately been realizing I'd like to take advantage of that feature myself.

Reply:
Chosen name is `inject_variance`.